wavelength_router/wavelength_router.proto

```diff
@@ -34,17 +34,17 @@ service WavelengthRouter {
   // the initiating client. A client that becomes disconnected during an
   // operation can reconnect, requesting the same operation, and receive status
   // updates this way. There is no means by which an adjustment that has been
-  // triggered by a client can be cancelled unless the CancelAdjustPSD RPC is
-  // used.
-  rpc AdjustPSD(AdjustPSDRequest) returns (stream AdjustPSDResponse) {}
+  // triggered by a client can be cancelled unless the CancelAdjustSpectrum
+  // RPC is used.
+  rpc AdjustSpectrum(AdjustSpectrumRequest) returns (stream AdjustSpectrumResponse) {}
```

Comment:
Do we need to consider add a new rpc rather than just replacing this one?  (is there an easier migration path?)

Reply:
Since there are not yet any implementations, I think this will be the cleanest way long term.

Reply:
After some additional discussion, I think we should consider adding a new RPC rather than just wholesale replacing.  We can mark the existing rpc or message fields as deprecated, and remove it in a subsequent revision.

Reply:
Sure, the RPCs are marked as deprecated. I can remove in a subsequent revision. Please let me know if this needs anything else.